src/MIDebugPackage/OpenFolderSchema.json

```diff
@@ -205,6 +205,13 @@
                       "type": "string"
                     }
                   },
+                  "pipeCmd": {
+                    "type": "array",
+                    "description": "Command line arguments passed to the pipe program to execute commands.",
```

Comment:
>Command line arguments passed to the pipe program to execute commands. [](start = 36, length = 70)

I think the description we have in the .xsd is better: `Arguments passed to the pipe executable to run a remote Unix (ex: bash or similar) shell command. This is used for running 'kill' on the remote system, or other commands that the debugger may need. If not specified, the debugger will do its best without the shell. But some features, such as setting breakpoints in run mode, may not work. This string should contain the string '{0}' which will be replaced with the command to execute.`

Reply:
Fixed.